### PR TITLE
fix(driver-adapters-wasm): cleanup

### PR DESCRIPTION
**Pull request by millsp**
My main intention was to close https://github.com/prisma/team-orm/issues/764, which was done mostly via the Cloudflare UI and also I did increase the wait period of a page deployment to reduce flakiness already before. So this is just a cleanup/fix/improvement for things I found along the way.

### CODE REVIEW

File: driver-adapters-wasm/neon-node-nextjs-edgemw/run.sh [hunk 1]
```diff
@@ -12,5 +12,6 @@ wait_for_host() {
 
 pnpm install
 pnpm next dev --port 3000 &
+echo $! > server.pid
```
Comment (millsp) [inline driver-adapters-wasm/neon-node-nextjs-edgemw/run.sh:16]: Caused the second step for custom output tests to fail.

File: driver-adapters-wasm/pg-cfpages-basic/functions/function.js [hunk 2]
```diff
@@ -2,9 +2,10 @@
 import { Prisma, PrismaClient } from '@prisma/client'
 import { Pool } from 'pg'
 import { PrismaPg } from '@prisma/adapter-pg'
+import { DATABASE_URL } from './dbUrl.js'
 
 export async function onRequest(context) {
-  const client = new Pool({ connectionString: context.env.DATABASE_URL })
+  const client = new Pool({ connectionString: DATABASE_URL })
```
Comment (millsp) [inline driver-adapters-wasm/pg-cfpages-basic/functions/function.js:9]: Now uses the postgresql database provided by ecosytem-tests instead.

File: driver-adapters-wasm/neon-node-nextjs-edgemw/run.sh (hunk 1 above)
Comment (Jolg42) [reply driver-adapters-wasm/neon-node-nextjs-edgemw/run.sh:15]: What do you mean? It's hard to understand without context for me at the moment 🤔

Comment (Jolg42) [inline driver-adapters-wasm/neon-node-nextjs-edgemw/run.sh:16]:
```suggestion
# create a `server.pid` file, which will be used in `finally.sh` to kill the process
echo $! > server.pid
```

**Review by Jolg42 — APPROVED** [review]

File: driver-adapters-wasm/pg-cfpages-basic/functions/function.js (hunk 2 above)
Comment (Jolg42) [inline driver-adapters-wasm/pg-cfpages-basic/functions/function.js:9]:
```suggestion
	// `DATABASE_URL` comes from `dbUrl.js` and its value is set by `prepare.sh`
  const client = new Pool({ connectionString: DATABASE_URL })
```

**Pull request merged**